examples/hello_world/example.toml

```diff
@@ -1,14 +1,13 @@
 name = "hello_world"
 
 [application]
+variant = "rust"
```

Comment:
(optional) Since you are adding an application variant, we may as well have multiple application variants in the same example manifest now. This would probably require making `modules` a field of `application`, which I think makes sense anyways.

Reply:
I actually think this is really nice. I don't like the fact that we hardcode "example_cpp.toml" in the runner, especially since I would like to have a "rust only" version of an example, and that would require yet another hardcoded version.

Reply:
Done